modules/nf-core/biohansel/main.nf

```diff
@@ -0,0 +1,45 @@
+process BIOHANSEL {
+    tag "$meta.id"
+    label 'process_low'
```

Comment:
Is this the correct default for this module? How much resources does it actually use?

Reply:
Can be multi threaded, so I selected the lowest with multiple cores.